docs/providers/aws/events/apigateway.md

```diff
@@ -771,6 +771,26 @@ functions:
                 id: true
 ```
 
+To map different values for request parameters, defines the `required` and `mappedValue` properties of the request parameter.
+
+```yml
+functions:
+  create:
+    handler: posts.post_detail
+    events:
+      - http:
+          path: posts/{id}
+          method: get
+          request:
+            parameters:
+              paths:
+                id: true
+              headers:
+                custom-header: context.requestId
+```
+
+For a list of acceptable values, see the [AWS Documentation](https://docs.aws.amazon.com/apigateway/latest/developerguide/request-response-data-mappings.html)
+
```

Comment:
Let's also list this settings in https://github.com/serverless/serverless/blob/94ff3b22ab13afc60fb4e672520b4db527ee0432/docs/providers/aws/guide/serverless.yml.md

(altough I believe we need to add `request` section in full, as it's missing)

Reply:
I've gone ahead and added what I could for the `request` section in commit 3b7675c

Reply:
Thank you!